### PR TITLE
[CV2-3249] track error

### PR DESCRIPTION
**Pull request by DGaffney**
https://meedan.atlassian.net/browse/CV2-3249

### CODE REVIEW

**Comment by skyemeedan** [comment]
I think it is helpful for reviewing to link to the ticket :-)  ... but still not much context.  Is this just trying to get more info into sentry for debugging the underlying issue?

File: app/main/lib/shared_models/video_model.py [hunk 1]
```diff
@@ -186,8 +187,8 @@ def search(self, task):
             files = self.get_fullpath_files(qualified_matches, False)
             try:
             	scores = tmkpy.query(self.tmk_file_path(video.folder, video.filepath),files,1)
-            except RuntimeError as e:
-            	print(e)
+            except Exception as err:
```
[the remark this answers is not in the document]
Comment (skyemeedan) [reply app/main/lib/shared_models/video_model.py:190]: Dear CodeClimate,  I was just going to say the same thing!  But if the point is to do it for logging, we could catch and re-raise the exception so we don't end up proceeding with a bad state?

**Comment by DGaffney** [comment]
> I think it is helpful for reviewing to link to the ticket :-) ... but still not much context. Is this just trying to get more info into sentry for debugging the underlying issue?

That's right - we're just trying to get more visibility, as the error we're seeing in sentry right now is an unhelpful one downstream of an error that's otherwise being swallowed currently.

File: app/main/lib/shared_models/video_model.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DGaffney) [reply app/main/lib/shared_models/video_model.py:190]: Thats the point my friend

[the remark this answers is not in the document]
Comment (DGaffney) [reply app/main/lib/shared_models/video_model.py:190]: my friend being codeclimate not @skyemeedan, who is also my friend incidentally

**Review by skyemeedan — APPROVED** [review]

**Review by computermacgyver — APPROVED** [review]

**Comment by computermacgyver** [comment]
Makes sense. I see in the full diff that we're catching the error, logging, and re-raising it. Someone should tell codeclimate to include more lines in the comment diff to give the full context :joy:

**Pull request merged**